src/ripple/rpc/tests/JSONRPC.test.cpp

```diff
@@ -1968,7 +1964,8 @@ class JSONRPC_test : public beast::unit_test::suite
             // 2: high mult, can queue, no pad
             Json::Value req;
             Json::Reader().parse(
-                "{ \"fee_mult_max\" : 1000, \"x-queue-okay\" : true \"tx_json\" : { } } ", req);
+                "{ \"fee_mult_max\" : 1000, \"x-queue-okay\" : true, "
+                    "\"tx_json\" : { } } ", req);
```

Comment:
Thanks for the suggestion. Funny thing is that I _know_ about raw strings, but I keep forgetting to use them.